### PR TITLE
init template v4

### PR DESCRIPTION
**Pull request by DyannT**
## Related Tickets:
- [#Ticket_ID](https://edu-redmine.sun-asterisk.vn/issues/xxxx)

## Notes Impacted Areas:
- N/A.

## Attachments:
- N/A.

### CODE REVIEW

**Comment by DyannT** [comment]
ready

File: resources/views/components/pages/follow/containerFollows.blade.php [hunk 1]
```diff
@@ -0,0 +1,98 @@
...
+                    <table class="table table-custom">
+                        <thead>
+                        <tr>
+                            <th scope="col">STT</th>
+                            <th scope="col">Avatar</th>
+                            <th scope="col">Name</th>
+                            <th scope="col">Role</th>
+                            <th scope="col">Status</th>
+                            <th scope="col">Time</th>
+                        </tr>
```
Comment (ghost) [inline resources/views/components/pages/follow/containerFollows.blade.php]: localization?

File: resources/views/components/pages/managerPost/containerManagerPost.blade.php [hunk 2]
```diff
@@ -0,0 +1,167 @@
...
+                            {{ __('Publish') }}
+                        </div>
+                        <div class="line"></div>
+                    </div>
+                    <div class="tab-content-custom">
+                        <div class="tab-pane active">
+                            <table class="table table-custom">
+                                <thead>
+                                <tr>
+                                    <th scope="col">ID1</th>
```
Comment (ghost) [inline resources/views/components/pages/managerPost/containerManagerPost.blade.php]: ID1?

**Comment by DyannT** [comment]
ready

**Pull request merged**